Simplenote/src/main/java/com/automattic/simplenote/viewmodels/ScopedViewModel.kt

```diff
@@ -0,0 +1,16 @@
+package com.automattic.simplenote.viewmodels
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.*
+import kotlin.coroutines.CoroutineContext
+
+abstract class ScopedViewModel(private val defaultDispatcher: CoroutineDispatcher) : ViewModel() {
```

Comment:
I know we use this approach in WPAndroid but I think you don't need this class (there are some historical reasons why it's used there). I think you can just use the plan `ViewModel` and the `viewModelScope.launch` here. In the unit tests you could replace the main dispatcher or inject it and call viewModelScope.launch(injectedDispatcher). I think we will eventually replace it in WPAndroid as well.

Reply:
Ok sounds good. Fixed in c11e2f2e680613700fdff1b2559b0da36fe07e9b

Reply:
I removed it for now. I will take a look when I need to use coroutines (mostly in `TagsActivity`).

Fixed in c11e2f2e680613700fdff1b2559b0da36fe07e9b.